apps/reaper/lib/reaper/data_extract/processor.ex

```diff
@@ -64,14 +64,16 @@ defmodule Reaper.DataExtract.Processor do
 
     wait_for_completion([producer_stage, validation_stage, schema_stage, load_stage])
 
+    messages_processed_count = messages_processed_count(cache_name)
```

Comment:
Why rearrange the messages_processed_count?

Reply:
Another symptom of re-doing things a few times and forgetting what all I did...